scripts/validate_docstrings.py

```diff
@@ -490,6 +514,13 @@ def validate_one(func_name):
         for param_err in param_errs:
             errs.append('\t{}'.format(param_err))
 
+    pep8_errs = [error for error in doc.validate_pep8()]
+    if pep8_errs:
+        errs.append('Errors in doctests')
```

Comment:
Not exactly "errors" I'd say:

```suggestion
        errs.append('Linting issues in doctests:')
```